tests/rt.conf

```diff
@@ -4,8 +4,8 @@
 
 COMPILE | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp                       |                                         | fv3 |
 RUN     | cpld_control_p8_mixedmode                                                                                               |                                         | fv3 |
-COMPILE | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp                                  |                                         | fv3 |
 RUN     | cpld_control_gfsv17                                                                                                     | - wcoss2.intel                          | fv3 |
```

Comment:
You'd need to remove the '- wcoss2.intel' here.

Reply:
I totally missed this.  Thanks for pointing it out @BrianCurtis-NOAA   I  also missed this when running my tests.  Let me re-check that this actually works before removing here.